core/src/main/java/io/aiven/klaw/config/ManageDatabase.java

```diff
@@ -181,6 +182,8 @@ private void loadStaticDataToDb() throws KlawException {
           defaultDataService.getDefaultTenant(KwConstants.DEFAULT_TENANT_ID));
     }
 
+    setDefaultEntitySequencesForTenant(KwConstants.DEFAULT_TENANT_ID);
```

Comment:
if someone is migrating from 2.4 to 2.5, this code will be executed too and the one defined in Migration2x5x0

Reply:
I have now made it so that they can run but they will check to see if they are already set or not.